docs/content/en/docs/resources/_index.md

```diff
@@ -18,6 +18,7 @@ description: "Additional resources to learn about Tetragon"
 | [Keeping your cluster safe from attacks with eBPF](https://www.youtube.com/watch?v=agN68U8x1go)                                                              | Jed Salazar & Natália Réka Ivánkó    | eBPF Summit                                       | 2021 |
 | [Uncovering a Sophisticated Kubernetes Attack in Real Time Part II.](https://www.oreilly.com/library/view/infrastructure-ops/0636920625377/video335775.html) | Jed Salazar & Natália Réka Ivánkó    | O'Reilly Superstream Series, Infrastructure & Ops | 2021 |
 | [Uncovering a Sophisticated Kubernetes Attack in Real-Time](https://www.youtube.com/watch?v=bohnofE_dvw)                                                     | Jed Salazar & Natália Réka Ivánkó    | KubeCon EU                                        | 2020 |
+| [eBPF-based, Kubernetes-native: observability & security with Tetragon](https://www.youtube.com/watch?v=xqKp6GfMuic)                                                     | Anna Kapuścińska    | TechSpot                                         | 2024 |
```

Comment:
please put it in the correct order, most recent up

Reply:
done